OMCompiler/Compiler/SimCode/SimCodeMain.mo

```diff
@@ -857,24 +857,19 @@ algorithm
         copyFiles(RuntimeSources.simrt_c_headers, source=install_include_omc_c_dir, destination=fmu_tmp_sources_dir);
         // The simrt C source files are installed to the folder specified by RuntimeSources.fmu_sources_dir. Copy them from there.
         copyFiles(RuntimeSources.simrt_c_sources, source=install_fmu_sources_dir, destination=fmu_tmp_sources_dir);
-
-        if varInfo.numLinearSystems > 0 or varInfo.numNonLinearSystems > 0 then
```

Comment:
Ah, is that condition unreliable as an indicator of needing the dgesv files?

Reply:
@bilderbuchi Yes the condition does not check if the model uses dgesv or other LAPACK sources. And also some of the LAPACK sources are missing in the source code and this will not solve all the issues related with missing LAPACK sources. I think the best to link the libOpenBlass.dll dynamically or statically in the cmake to solve the issue. And also i found that copying the whole LAPACK sources will be huge.